src/dstack/_internal/core/models/runs.py

```diff
@@ -239,6 +240,10 @@ class JobRuntimeData(CoreModel):
     # None if data is not yet available (on vm-based backends and ssh instances)
     # or not applicable (container-based backends)
     ports: Optional[dict[int, int]] = None
+    # List of volumes used by the job
+    volume_names: Optional[list[str]] = None  # None for backward compalibility
+    # Virtual shared offer. None if the instance is not shared.
+    offer: Optional[InstanceSharedOffer] = None
```

Comment:
Maybe it's more useful to always store the offer used by the job, not just when it was shared?

Reply:
Changed in https://github.com/dstackai/dstack/pull/2253/commits/8b555f7a6ad65afbed244e3d1e76359bdbbcbbf9